react/components/ProductName/index.js

```diff
@@ -48,7 +48,7 @@ class ProductName extends Component {
   }
 
   static Loader = (loaderProps = {}) => (
-    <div className={`${productName.productNameContainer} ${productName.productNameLoader} ${loaderProps.className}`}>
+    <div className={productName.productNameContainer, productName.productNameLoader, loaderProps.className}>
```

Comment:
You're missing the usage of `classnames` funciton here, this code is only applying the classes of `loaderProps.className`